tests/integration/telemetry/wasmplugin/wasm_modules/header_injector/src/lib.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use log::trace;
```

Comment:
can we write this demo in go, most of developer in this org farmiliar with go than rust.

Reply:
Currently, [proxy-wasm](https://github.com/orgs/proxy-wasm/repositories) only supports Rust and C++, so I went to `rust`. I thought that, since this is just very small code piece, the language is not problematic.

Of course, there is a way to use `Go`: [go-sdk](https://github.com/tetratelabs/proxy-wasm-go-sdk) from Tetrate.

@howardjohn @douglas-reid Could you please give your thoughts?

Reply:
IMHO it's more important to keep this stable. C++ is the most stable SDK. Go is not really idiomatic since Wasm doesn't support garbage collection.

Reply:
IMHO, go-sdk > c++ sdk > rust sdk.

Reply:
I moved c++ since @kyessenov mentioned, most stable SDK seems to be c++ version at now.